docs/alert_check_scheduled_jobs.md

```diff
@@ -15,3 +15,35 @@ It is possible that a given travel advice item or medical alert might be updated
 ## Dependencies
 
 This system depends on the Search API structure, so radical changes to the search API's results format may cause problems.
+
+## Manual Checking
+
+There is a rake task to create a subscriber to both the medical and travel alerts. In production this account will persist, but if you want to use it in integration you will need to run the task before testing.
```

Comment:
Is this due to the prod addresses getting anonymised(?) - if so could add it here for extra context